### ISSUE TITLE
Check download that fails at the end

### ISSUE THREAD

**Issue by Cyklet** [Cyklet/VidLoader#31]
Some of the videos that are almost downloaded are failling:
```swift
Error Domain=CoreMediaErrorDomain Code=-16240 "(null)" UserInfo={_NSURLErrorRelatedURLSessionTaskErrorKey=(
    "BackgroundAVAssetDownloadTask <B794ED94-1B6A-4717-9A66-FEA0C2DF3355>.<2>"
), _NSURLErrorFailingURLSessionTaskErrorKey=BackgroundAVAssetDownloadTask <B794ED94-1B6A-4717-9A66-FEA0C2DF3355>.<2>}
```

**Comment by tridorje** [comment]
@Cyklet 
Can you fix this :D

**Comment by Cyklet** [comment]
Fixed with #29
